public/angular/templates/api-settings-modal.html

```diff
@@ -14,12 +14,6 @@ <h5 class="modal-title">
         <li translate>api_settings_modal.twitter.p1</li>
         <li translate>api_settings_modal.twitter.p2</li>
       </ol>
-      <ol ng-if="item === 'facebook'">
-        <li translate>api_settings_modal.facebook.p1</li>
-        <li translate>api_settings_modal.facebook.p2</li>
-        <li translate>api_settings_modal.facebook.p3</li>
-        <li translate>api_settings_modal.facebook.p4</li>
-      </ol>
```

Comment:
Clearly the old Facebook help was obsolete here, but might we replace it with CT help instead of removing it entirely?

Reply:
done!